Drop-In/src/main/java/com/braintreepayments/api/DropInActivity.java

```diff
@@ -287,15 +286,21 @@ private void finishDropInWithPendingResult(DropInExitTransition transition) {
     }
 
     private void updateVaultedPaymentMethodNonces(boolean refetch) {
-        if (clientTokenPresent) {
-            dropInClient.getVaultedPaymentMethods(this, (vaultedPaymentMethods, error) -> {
-                if (vaultedPaymentMethods != null) {
-                    dropInViewModel.setVaultedPaymentMethods(vaultedPaymentMethods);
-                } else if (error != null) {
-                    onError(error);
+        dropInClient.getAuthorization(new AuthorizationCallback() {
+            @Override
+            public void onAuthorizationResult(@Nullable Authorization authorization, @Nullable Exception authorizationError) {
```

Comment:
We should handle this authorizationError too

Reply:
This one is kind of covered actually. If authorization is `null` then `instanceof ClientToken` will return false.